frontend/awx/resources/templates/components/Sparkline.tsx

```diff
@@ -40,7 +43,11 @@ export const Sparkline = ({ jobs }: { jobs: SummaryFieldRecentJob[] | undefined
     <Tooltip position="top" content={generateTooltip(job)} key={job.id}>
       <Link
         aria-label={t(`View job ${job.id}`)}
-        to={`/jobs/${JOB_TYPE_URL_SEGMENTS[job.type]}/${job.id}/output`}
+        to={
+          getJobOutputUrl(job as UnifiedJob)
+            ? getJobOutputUrl(job as UnifiedJob)
+            : `/jobs/${JOB_TYPE_URL_SEGMENTS[job.type]}/${job.id}/output`
```

Comment:
Maybe you can try to use the null coalescing operator here. That way the function to get the url isn't called twice.
```suggestion
          getJobOutputUrl(job as UnifiedJob) ?? `/jobs/${JOB_TYPE_URL_SEGMENTS[job.type]}/${job.id}/output`
```

Reply:
if `getJobOutputUrl` is `undefined` or `null` it'll default to using  `/jobs/${JOB_TYPE_URL_SEGMENTS[job.type]}/${job.id}/output`. If `getJobOutputUrl` returns a non null value, then it will use that.

Reply:
I agree and like that better, but it gets a component test failure in InventoryHostDetails. I can try to make a change or two to see if I can fix that problem.

Reply:
For some reason there is a test failure when using the null coalescing operator, but was able to reduce the number of calls by defining jobOutputUrl and then using its value for the ternary operation. Checked with @prat98 and resolving this change :)